README.md

```diff
@@ -1,13 +1,9 @@
-# jumpstart
+# Jumpstart
 
-Project generator for various languages and frameworks
+Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.  Simply follow the instructions below to get started. 
+##### *It is actively under development to support different frameworks and languages, for updates please star the repository
```

Comment:
```suggestion
> [!IMPORTANT]  
> Jumpstart is under active development to support different frameworks and languages, for updates please star (or watch) the repository.
```

Reply:
For reference: https://blog.jakelee.co.uk/github-alert-experiments/

Reply:
done